ironfish-rust/src/merkle_note.rs

```diff
@@ -162,6 +166,30 @@ impl MerkleNote {
         })
     }
 
+    /// Load a MerkleNote from the given reader, skipping some expensive validity checks on the
+    /// input.
+    ///
+    /// This method is faster than [`read()`], but it requires trusted or pre-validated input. It
+    /// might panic if the input is not properly encoded.
```

Comment:
By panic do you mean return error result?

Reply:
I recall there was some panic behavior somewhere, but I can't find it anymore. I guess that, when I wrote this line, the code was a bit different. I'm going to reword this comment.